Applications/opensense/opensense.cpp

```diff
@@ -258,13 +258,12 @@ void PrintUsage(const char *aProgName, ostream &aOStream)
     aOStream << "-PropertyInfo, -PI                      Print help information for properties in setup files.\n";
     aOStream << "-ReadX, -RX  directory settings.xml     Parse Xsens exported files from directory using settingsFile.xml.\n";
     aOStream << "-ReadA, -RA  datafile.csv settings.xml  Parse single csv file provided by APDM using specified settingsFile.xml.\n";
-    aOStream << "-Calibrate, -C IMUPlacer_setup.xml      Calibrate the model specified in the IMUPlacer_setup.xml file by registering\n";
-    aOStream << "                                        IMU frames whose orientations in the sensor world frame are described by quaternions file\n";
-    aOStream << "                                        specified in IMUPlacer_setup.xml and assuming the model's default pose \n";
-    aOStream << "                                        is the calibration pose. The resultant model with IMU frames registered \n";
-    aOStream << "                                        is written to file if output_model_file is specified in IMUPlacer_setup.xml.";
-    aOStream << "                                        Additional options can be described in IMUPlacer_setup.xml\n";
-    aOStream << "                                        to perform heading correction.\n";
+    aOStream << "-Calibrate, -C IMUPlacer_setup.xml      Place IMUs on the model that is specified in the IMUPlacer_setup.xml file.\n";
+    aOStream << "                                        The model is positioned in its default pose, IMUs are then registered to the model according to\n ";
```

Comment:
There should be a period after "The model is positioned in its default pose" rather than a comma. Otherwise the content looks good to me.